### PR TITLE
Adds KCS article to docs to address hpa unmanaged issue

### PR DESCRIPTION
**Pull request by stevsmit**

### CODE REVIEW

File: modules/operator-unmanaged-hpa.adoc [hunk 1]
```diff
@@ -6,21 +6,105 @@ Horizontal Pod Autoscalers (HPAs) are now included with the `Clair`, `Quay`, and
...
+    managed: true
+    overrides: 
+      replicas: null <1>
 # ...
+----
+<1> After setting `replicas: null` in your `QuayRegistry` CRD, a new replica set might be generated because the deployment manifest of the `Quay` app is changed with `replicas: 1`. 
+
+.Verification
+
+. Create a customized `HorizontalPodAutoscalers` CRD and increase the `minReplicas` amount to a higher value, for exampe, `6`:
```
Comment (SeanZhao-redhat) [inline modules/operator-unmanaged-hpa.adoc:51]: for exampe, `6` --> suggest to: for exampe, `3`, as below examples all shows 3 replicas

File: modules/operator-unmanaged-hpa.adoc [hunk 2]
```diff
@@ -6,21 +6,105 @@ Horizontal Pod Autoscalers (HPAs) are now included with the `Clair`, `Quay`, and
...
+apiVersion: autoscaling/v2
+metadata:
+  name: quay-registry-quay-app
+  namespace: quay-enterprise
+spec:
+  scaleTargetRef:
+    kind: Deployment
+    name: quay-registry-quay-app
+    apiVersion: apps/v1
+  minReplicas: 6
```
Comment (SeanZhao-redhat) [inline modules/operator-unmanaged-hpa.adoc:65]: same, suggest to : `minReplicas: 3`

File: modules/operator-unmanaged-hpa.adoc [hunk 3]
```diff
@@ -6,21 +6,105 @@ Horizontal Pod Autoscalers (HPAs) are now included with the `Clair`, `Quay`, and
 
 As HPA is configured by default to be managed, the number of `Clair`, `Quay`, and `Mirror` pods is set to two. This facilitates the avoidance of downtime when updating or reconfiguring {productname} through the Operator or during rescheduling events.
 
+[NOTE]
+====
+There is a known issue when disabling the `HorizontalPodAutoscaler` component and attempting to edit the HPA resource itself and increase the value of the `minReplicas` field. When attempting this setup, `Quay` application pods are scaled out by the unmanaged HPA and, after 60 seconds, the replica count is reconciled by the {productname} Operator. As a result, HPA pods are continuously created and then removed by the Operator.
```
Comment (SeanZhao-redhat) [inline modules/operator-unmanaged-hpa.adoc:11]: "There is a known issue xxxx" ->
Question: 
can we list the issue number here? I mean something like:  There is a known issue [PROJQUAY-6474](https://issues.redhat.com/browse/PROJQUAY-6474)
how do we usually do

File: modules/operator-unmanaged-hpa.adoc [hunk 4]
```diff
@@ -6,21 +6,105 @@ Horizontal Pod Autoscalers (HPAs) are now included with the `Clair`, `Quay`, and
...
++
+[source,terminal]
+----
+$ oc get hpa
+----
++
+[source,terminal]
+----
+NAME                     REFERENCE                           TARGETS            MINPODS   MAXPODS   REPLICAS   AGE
+quay-registry-quay-app   Deployment/quay-registry-quay-app   67%/90%, 54%/90%   2         20        3          51m
```
Comment (SeanZhao-redhat) [inline modules/operator-unmanaged-hpa.adoc:109]: the value of MINPODS should be 3, not 2

File: modules/operator-unmanaged-hpa.adoc (hunk 1 above)
Comment (stevsmit) [reply modules/operator-unmanaged-hpa.adoc:51]: Changed to 3

File: modules/operator-unmanaged-hpa.adoc (hunk 2 above)
Comment (stevsmit) [reply modules/operator-unmanaged-hpa.adoc:65]: Changed to 3

File: modules/operator-unmanaged-hpa.adoc (hunk 3 above)
Comment (stevsmit) [reply modules/operator-unmanaged-hpa.adoc:11]: Yes. Done.

File: modules/operator-unmanaged-hpa.adoc (hunk 4 above)
Comment (stevsmit) [reply modules/operator-unmanaged-hpa.adoc:109]: Fixed.

**Pull request merged**